docs/resources/system_schema.md

```diff
@@ -3,14 +3,12 @@ subcategory: "Unity Catalog"
 ---
 # databricks_system_schema Resource
 
--> This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html).
```

Comment:
need to add this back

Reply:
it depends on the table. In general they are GA: https://learn.microsoft.com/en-us/azure/databricks/release-notes/product/2024/september#system-tables-are-now-generally-available